.gitignore

```diff
@@ -7,6 +7,7 @@
 .idea/
 .nixos-test-history
 .vscode/
+.helix/
```

Comment:
I would add that this shouldn't be part of this PR and added into the user global's `.gitignore` file.

Reply:
I think it's reasonable to have this, esp. since we also have it for other editors like .idea, .vscode, etc... but yes I agree it would have been more transparent if we moved it into a separate pr.